packages/eslint-plugin-sui/docs/rules/factory-pattern.md

```diff
@@ -0,0 +1,35 @@
+# Ensure that our classes are using the convetion of has a static create method as factory. (`factory-pattern`)
+
+Please describe the origin of the rule here.
```

Comment:
❓

Reply:
I guess contents of this file are missing for now.

Reply:
yep! We have to review the documentation of each rule, when we work on it